models/instruct_pix2pix.py

```diff
@@ -0,0 +1,99 @@
+import datetime
+import json
+import os
+import shutil
+import PIL
+import dtlpy as dl
+import torch
+from diffusers import StableDiffusionInstructPix2PixPipeline, EulerAncestralDiscreteScheduler
+
+STREAM_URL = r"https://gate.dataloop.ai/api/v1/items/{}/stream"
+
+
+def create_folder(folder):
+    if os.path.exists(folder):
+        shutil.rmtree(folder, ignore_errors=True)
+    os.mkdir(folder)
+
+
+class HuggingAdapter:
+    def __init__(self, configuration):
+        self.model_name = configuration.get("model_name")
+
+        self.model = StableDiffusionInstructPix2PixPipeline.from_pretrained("timbrooks/instruct-pix2pix",
+                                                                            torch_dtype=torch.float16,
+                                                                            safety_checker=None)
+        self.model.to("cuda")
+        self.model.scheduler = EulerAncestralDiscreteScheduler.from_config(self.model.scheduler.config)
+        self.results_local_path = "instruct_pix2pix_results"
+        create_folder(self.results_local_path)
+
+    def prepare_item_func(self, item: dl.Item):
+        buffer = json.load(item.download(save_locally=False))
+        prompts = buffer["prompts"]
+        ready_prompts = []
+        for prompt_key, prompt_content in prompts.items():
+            image_buffer, prompt_text = None, None
+            for prompt_part in prompt_content:
```

Comment:
Now that I tested some other models, I think the same thing applies here. It should be `prompt_content.value()` instead